bus.js

```diff
@@ -0,0 +1,214 @@
+    welcome.style.display = 'none';
+    var restart = document.getElementById('restart');
+    restart.style.display = 'block';
+    displayChart();
+    displayChartTwo();
+    getTable();
+    try {
+      localStorage.photos = JSON.stringify(photos);
+    } catch (error) {
+      console.log('something went wrong', error);
+    }
+  }
+}
+
+function displayChart() {
+  photos = photos.concat(photosOnSecondToLastScreen);
+  photos = photos.concat(photosOnScreen);
+  photos = photos.concat(photosOnPreviousScreen);
+  var chartLabel = [];
+  var clicks = [];
+  var displays = [];
+  for (var i = 0; i <photos.length; i++) {
+    chartLabel.push(photos[i].name);
+    clicks.push(photos[i].clicks);
+    displays.push(photos[i].displayed);
+  }
+
+  var canvas = document.getElementById('chart-canvas1');
+  var ctx = canvas.getContext('2d');
+  var data = {
+    labels: chartLabel,
+    datasets:[{
+      label: 'Clicks',
+      backgroundColor:'#42826C',
+      borderColor: '#002F32',
+      borderWidth: 1,
+      data: clicks},
+    {
+      label: 'Displayed',
+      backgroundColor: '#A5C77F',
+      borderColor: '#002F32',
+      borderWidth: 1,
+      data: displays,}
+    ]
+  };
+  canvas.height = '500';
+  canvas.width = '500';
+  var myBarChart = new Chart(ctx, {
+    type: 'bar',
+    data: data,
+  });
+}
+function displayChartTwo() {
+  var chartLabel = [];
+  var percent= [];
+  for (var i = 0; i <photos.length; i++) {
+    chartLabel.push(photos[i].name);
+    percent.push(Math.floor((photos[i].clicks/photos[i].displayed)*100));
+  }
+  var canvas = document.getElementById('chart-canvas2');
+  var ctx = canvas.getContext('2d');
+  Chart.defaults.global.defaultFontColor = '#000000';
+  var data = {
+    labels: chartLabel,
+    datasets:[{
+      label: '% of Clicks when Viewed',
+      backgroundColor:'#002F32',
+      borderColor: '#42826C',
+      borderWidth: 1,
+      data: percent,}
+    ]
+  };
+  canvas.height = '500';
+  canvas.width = '500';
+  var myBarChart = new Chart(ctx, {
+    type: 'bar',
+    data: data,
+  });
+}
+
+function getTable(){
+
+  var result = document.getElementById('table-data');
+  var table = document.createElement('table');
+  table.id = table;
+  var titleRow = document.createElement('tr');
+  var titleItem = document.createElement('th');
+  titleItem.textContent = 'Item';
+  var titleViews = document.createElement('th');
+  titleViews.textContent = 'Views';
+  var titleClicks = document.createElement('th');
+  titleClicks.textContent = 'Clicks';
+  var titlePercetage = document.createElement('th');
+  titlePercetage.textContent = '% of Clicks When Viewed';
+  var titleRecomended = document.createElement('th');
+  titleRecomended.textContent = 'Recomended';
+  result.appendChild(table);
+  table.appendChild(titleItem);
+  table.appendChild(titleViews);
+  table.appendChild(titleClicks);
+  table.appendChild(titlePercetage);
+  table.appendChild(titleRecomended);
```

Comment:
Consider an array to hold your column titles and a for loop to iterate through as you are repeating the same line of code only changing the value used.